src/dbnode/storage/bootstrap/bootstrapper/persist.go

```diff
@@ -285,7 +291,12 @@ func BuildBootstrapIndexSegment(
 
 	compactor.Lock()
 	defer compactor.Unlock()
-	seg, err := compactor.Compactor.CompactUsingBuilder(b.Builder(), nil)
+	seg, err := compactor.Compactor.CompactUsingBuilder(b.Builder(), nil, mmap.ReporterOptions{
+		Context: mmap.Context{
+			Name: bootstrappedIndexBlockMmapName,
+		},
+		Reporter: mmapReporter,
+	})
```

Comment:
Yeah this is nicer when the entire `mmap.ReporterOptions{...}` is set just before use.